src/Utils.js

```diff
@@ -7,16 +7,31 @@ import {
 } from './react-compat';
 import {
   REACT013,
-  REACT014,
+  REACT15,
 } from './version';
 
+function internalInstanceKey(node) {
+  return Object.keys(Object(node)).filter(key => key.match(/^__reactInternalInstance\$/))[0];
```

Comment:
Really?

![cartoon drawing of a sad fox](https://cloud.githubusercontent.com/assets/6820/25299953/360fab9a-26ba-11e7-8216-3db879b5dd36.png)